puppeteer-tests/src/comments/comment-utils.ts

```diff
@@ -1,47 +1,61 @@
-import type { Page } from 'puppeteer'
+import type { ElementHandle, Page } from 'puppeteer'
 import type { UtopiaPuppeteerBrowser } from './test-utils'
 
 export const TIMEOUT = 120000
+export const PUPPETEER_TIMEOUT = 30000
 
 const BRANCH_NAME = process.env.BRANCH_NAME ? `&branch_name=${process.env.BRANCH_NAME}` : ''
 const BASE_URL = process.env.BASE_URL ?? 'http://localhost:8000'
 
+export async function getElementWithSelector(
+  page: Page,
+  selector: string,
+): Promise<ElementHandle<Element> | null> {
+  console.info('waiting for element with selector', selector)
+  const element = await page.waitForSelector(selector)
+  return element!
```

Comment:
Is this forcing the result to not be null? When the result type allows that.

Reply:
good catch, I wanted it force it not to be null, I'll update the return type

Reply:
removed the `| null` in https://github.com/concrete-utopia/utopia/pull/4789/commits/0dc014050e525ee8bc39999483b35a2a5e9bbf16

Reply:
I'd recommend using `forceNotNull` as I believe the `!` just coerces the type not the value itself.